pkg/ec/error.go

```diff
@@ -137,6 +138,22 @@ func NewErrNotImplemented() *ControllerError {
 	return NewControllerError(http.StatusNotImplemented)
 }
 
+// IsRetryable returns true and the retry delay if ANY errors eminating from the supplied error
+// is an ec.ControllerError that is Retryable, and false otherwise.
+func IsRetryable(err error) (bool, time.Duration) {
+
```

Comment:
extra line

Reply:
I prefer this; usually the top is for variables; then a newline, then the code